scalafmt-core/shared/src/main/scala/org/scalafmt/rewrite/ExpandImportSelectors.scala

```diff
@@ -21,7 +21,12 @@ class ExpandImportSelectors(implicit ctx: RewriteCtx) extends RewriteSession {
 
   override def rewrite(tree: Tree): Unit =
     tree match {
-      case q"import ..$imports" =>
+      case stat: ImportExportStat =>
+        val imports = stat.importers
+        val keyword = stat match {
+          case _: Export => "export"
```

Comment:
is there a KwImport that can be used instead?

Reply:
Good idea, changed to `val keyword = stat.tokens.head.text`

Reply:
I mean, I went a bit further here, since the first token is either `export` or `import`, so this works nicely.